fvtest/tril/tril/compiler_util.hpp

```diff
@@ -49,6 +50,28 @@ static TR::DataTypes getTRDataTypes(const std::string& name) {
    }
 }
 
+/**
+ * @brief Return a parsed array of DataTypes from a node with an 
+ *        "args" list.
+ * @param node is the node being processed.
+ * @return the std::vector of TR::DataTypes corresponding to the 
+ *         args=[Type1,Type2,...] attached to the ASTNode. 
+ */
+static std::vector<TR::DataTypes> parseArgTypes(const ASTNode* node) { 
+
+   std::vector<TR::DataTypes> argTypes;
+   auto argTypesArg = node->getArgByName("args");
+   if (argTypesArg != nullptr) {
```

Comment:
I hate having to make this comment, but please use `NULL` instead of `nullptr` (here and bellow) as the latter is not supported by all our build compilers.

Reply:
✅